.travis.yml

```diff
@@ -0,0 +1,64 @@
+language: cpp
+
+cache:
+  directories:
+    - ${TRAVIS_BUILD_DIR}/deps/cmake
+
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+      addons:
+        apt:
+          sources: ['ubuntu-toolchain-r-test']
+          packages: ['gcc-5', 'g++-5', 'xsltproc', 'libgtest-dev']
+      env:
+        - COMPILERCC=gcc-5
+        - COMPILERCXX=g++-5
+
+    - os: linux
+      dist: trusty
+      compiler: clang
+      addons:
+        apt:
+          sources: ['llvm-toolchain-trusty-4.0']
+          packages: ['clang-4.0', 'xsltproc', 'libgtest-dev']
+      env:
+        - COMPILERCC=clang-4.0
+        - COMPILERCXX=clang++-4.0
+
+
+    - os: osx
+      compiler: gcc
+      env:
+        - COMPILERCC=gcc
+        - COMPILERCXX=g++
+
+    - os: osx
+      compiler: clang
+      env:
+        - COMPILERCC=clang
+        - COMPILERCXX=clang++
+
+install:
+  - DEPS_DIR="${TRAVIS_BUILD_DIR}/deps"
+  - mkdir -p "${DEPS_DIR}" && cd "${DEPS_DIR}"
+  ############################################################################
+  # Install a recent CMake (unless already installed on OS X)
+  ############################################################################
+  - |
+    if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then
+      if [[ -z "$(ls -A ${DEPS_DIR}/cmake/bin)" ]]; then
+        CMAKE_URL="https://cmake.org/files/v3.8/cmake-3.8.0-rc2-Linux-x86_64.tar.gz"
+        mkdir -p cmake && travis_retry wget --no-check-certificate --quiet -O - "${CMAKE_URL}" | tar --strip-components=1 -xz -C cmake
+      fi
+      export PATH="${DEPS_DIR}/cmake/bin:${PATH}"
+    else
+      if ! brew ls --version cmake &>/dev/null; then brew install cmake; fi
+    fi
+
+script:
+  - cd "${TRAVIS_BUILD_DIR}"
+  - mkdir build; cd build
+  - export CC=$COMPILERCC; export CXX=$COMPILERCXX
+  - cmake .. && make && ctest
```

Comment:
instead write:
`cmake .. && make && CTEST_OUTPUT_ON_FAILURE=1 ctest`
so we'll able to see tests fails logs